src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Great job. please kindly ensure that the App.js component should be class-based

Reply:
Well resolved 👍🏽